src/Contao/SessionStorage.php

```diff
@@ -123,7 +158,7 @@ public function all()
     public function replace(array $attributes)
     {
         $this->load();
-        $this->attributes = \array_merge($this->attributes, $attributes);
+        $this->attributes = $attributes;
```

Comment:
This change makes the call of `$this->load();` obsolete, why did you do this?

Reply:
I have rework this

Reply:
a90cf59f